core/services/scheduler.go

```diff
@@ -193,7 +193,7 @@ func (ot *OneTime) RunJobAt(initiator models.Initiator, job models.JobSpec) {
 			return
 		}
 
-		if err := ot.Store.MarkRan(&initiator, true); err != nil {
```

Comment:
I don't understand this change. Anything wrong with references to orm.Model instances?

Reply:
Can't pass by pointer because the pointed object is shared amongst multiple goroutines. Easier to simply copy (and more idiomatic anyway, copying structs is cheap)

Reply:
Luckily, it has no pointer attributes that are likely to be mutated. But it's a good point.